hydromt_fiat/api/hydromt_fiat_vm.py

```diff
@@ -155,9 +155,13 @@ def run_hydromt_fiat(self):
             # Buildings and roads are set up
             full_gdf = self.fiat_model.exposure.get_full_gdf(exposure_db)
             buildings_gdf = full_gdf.loc[full_gdf["Primary Object Type"] != "roads"]
-            roads_gdf = full_gdf.drop(["SVI", "SVI_key_domain"], axis=1).loc[
-                full_gdf["Primary Object Type"] == "roads"
-            ]
+            if "SVI" in full_gdf.columns and "SVI_key_domain" in full_gdf.columns:
```

Comment:
is there a chance that only one of them is present in the columns? 
If yes, wouldn't something like:
 svi_cols = [col for col in full_gdf.columns if "SVI" in col]

if len(svi_cols)>0:
.....

Reply:
the SVI functionality adds both columns by default so the case that only one column would be added should not appear.